src/Raven.Server/Config/Categories/AiConfiguration.cs

```diff
@@ -29,4 +29,19 @@ public sealed class AiConfiguration : ConfigurationCategory
     [MinValue(1)]
     public int EmbeddingsMaxConcurrentBatches { get; set; }
 
+    [Description("Instruction text prepended to the serialized conversation when requesting a summary.")]
+    [DefaultValue(@"Summarize the following AI conversation into a concise, linear narrative that retains all critical information. Ensure the summary:
+- Includes key identifiers, usernames, timestamps, and any reference codes
+- Preserves the original intent of both the user and the assistant in each exchange
+- Reflects decisions made, suggestions given, preferences expressed, and any changes in direction
+- Captures tone when relevant (e.g., sarcastic, formal, humorous, concerned)
+- Omits general filler or small talk unless it contributes to context or tone Format the output in a structured manner (such as bullet points or labeled sections) suitable for fitting into a limited context window. Do not discard any information that contributes to understanding the conversation's flow and outcome.")]
+    [ConfigurationEntry("Ai.Agent.Summarization.SystemPrompt", ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
SystemPrompt => SummarizationTaskBeginningPrompt

also shoulndt this be: Ai.Agent.Trimming.Summarization.SummarizationTaskBeginningPrompt

Reply:
Done here:
https://github.com/ravendb/ravendb/pull/20973